### DISCUSSION TITLE
Connection with Wahoo Kickr

### DISCUSSION THREAD

**Discussion by michaelfisher44** [cagnulein/qdomyos-zwift#1397] [Ideas]
Hi, does QZ enable connection to Zwift from a Wahoo Kickr? I have loads of issues on Zwift with spinning out on flats and downhills and they seem to be refusing to add a setting to increase base resistance (despite popular demand). I heard that QZ has a setting like this and can be used to connect to Zwift (which would seem far more sensible than me having to buy a new bike with bigger gearing just for Zwift!). I went to download it but didn't see the Kickr listed as one of the Smart trainers that it connects to (I might be being blind though). Does this work with the Kickr? If so that would be great!

**Comment by cagnulein** [comment]
hi @michaelfisher44 yes it should already work! In case it doesn't work out of the box, please let me know and I will fix immediately! Let me know!

**Reply by michaelfisher44** [reply]
Awesome, thanks for the prompt response!
